general_configuration/_topics/configuration.md

```diff
@@ -3080,9 +3080,33 @@ from the global copy.
 
 8.  Click **Save**.
 
-#### Backing Up and Restoring a Database
-
-##### Running a Single Database Backup
+#### Backing Up and Restoring {{ site.data.product.title }} in a Kubernetes Based Environment (podified)
+
+**Prerequisites:**
+   - Velero and Restic must be installed and working in your cluster. For more information on Velero, see the [Velero documentation](https://velero.io/docs/v1.5/). For more information on Restic, see the [Restic documentation](https://restic.readthedocs.io/en/stable/) 
+
+**Note:** The operator adds labels and/or annotations to everything that needs to be backed up. You can easily and quickly backup only what is required. The operator will add annotations for restic volume backups for example, `backup.velero.io/backup-volumes: miq-pgdb-volume`.
+
+1. The backup label key can be configured by adding the following to the CR. The default applies the label `manageiq.org/backup=t` if nothing else is specified.
```

Comment:
Same here, but we probably want to make manageiq.org/backup=t a config parameter

Reply:
@bdunne when you say a config parameter you mean put it in https://github.com/ManageIQ/manageiq-documentation/blob/master/_data/product.yml ?